frontend/src/components/common/form/RadioGroup.tsx

```diff
@@ -0,0 +1,135 @@
+  /** The form component label to display after the checkbox */
+  postLabel?: string;
+  /** The underlying HTML element to use when rendering the FormControl */
+  as?: React.ElementType;
+  /** Short hint that describes the expected value of an <input> element */
+  placeholder?: string;
+  /** Adds a custom class to the input element of the <Input> component */
+  className?: string;
+  /** Whether the field is required. Makes the field border blue. */
+  required?: boolean;
+  /** Specifies that the HTML element should be disabled */
+  disabled?: boolean;
+  /** Use React-Bootstrap's custom form elements to replace the browser defaults */
+  custom?: boolean;
+  /** style to use for the formgroup wrapping the inner element */
+  innerClassName?: string;
+  /** override the display of the component, default is checkbox. Select radio to display this checkbox as two radio buttons. */
+  type?: string;
+  /** Optional tool tip message to add to checkbox */
+  toolTip?: string;
+  /** id for tooltip */
+  toolTipId?: string;
+};
+
+// only "field" is required for <RadioGroup>, the rest are optional
+export type RadioGroupProps = FormCheckProps & OptionalAttributes & RequiredAttributes;
+
+/**
+ * Formik-connected <RadioGroup> form control
+ */
+export const RadioGroup = ({
+  field,
+  radioValues,
+  label,
+  postLabel,
+  as: is, // `as` is reserved in typescript
+  placeholder,
+  className,
+  innerClassName,
+  required,
+  disabled,
+  type,
+  custom,
+  toolTip,
+  toolTipId,
+  ...rest
+}: RadioGroupProps) => {
+  const { values, errors, touched, handleBlur, handleChange } = useFormikContext();
+  const touch = getIn(touched, field);
+  const activeValue = getIn(values, field);
+  const error = getIn(errors, field);
+  return (
+    <StyledRadioGroup
+      controlId={`input-${field}`}
+      className={classNames(!!required ? 'required' : '', className)}
+    >
+      {!!label && (
+        <Form.Label>
+          {label}
+          {!!toolTip && <TooltipIcon toolTipId={toolTipId!} toolTip={toolTip} />}
+        </Form.Label>
+      )}
+      <div className="radio-group">
+        {radioValues.map(({ radioLabel, radioValue }, index) => (
+          <InlineFlexDiv key={`field-${radioValue}-${index}`}>
+            <Form.Check
+              id={`input-${radioValue}`}
+              name={field}
+              className={innerClassName}
+              required={required}
+              disabled={disabled}
+              custom={custom}
+              isInvalid={!!touch && !!error}
+              type="radio"
+              {...rest}
+              value={radioValue}
+              placeholder={placeholder}
+              checked={activeValue === radioValue}
+              onChange={handleChange}
+              onBlur={handleBlur}
+            />
+            <Form.Label className="form-check-label">{radioLabel}</Form.Label>
+          </InlineFlexDiv>
+        ))}
+      </div>
+      {!!postLabel && !!required && (
+        <>
```

Comment:
q: Since is a single child, is the <> necessary?

Reply:
no, you're correct. I've removed it.